pages/add-clinic.vue

```diff
@@ -141,8 +136,24 @@ export default {
         alert("Please fill out all of the fields!");
       }
     },
-    validate() {
-      this.$refs.form.validate();
+    errorMessages(field, name) {
+      if (field.$dirty && field.$invalid) {
```

Comment:
if you invert this it will be easier to read. You won't have to nest the whole block of logic.

Reply:
you can also get rid of the `else` when you do this

Reply:
Inverted and changed the condition to `field.$error`